src/openforms/config/models.py

```diff
@@ -136,8 +136,83 @@ class GlobalConfiguration(SingletonModel):
         ),
     )
 
+    # analytics/tracking
+    gtm_code = models.CharField(
+        _("Google Tag Manager code"),
+        max_length=50,
+        blank=True,
+        help_text=_(
+            "Typically looks like 'GTM-XXXX'. Supplying this installs Google Tag Manager."
+        ),
+    )
+    ga_code = models.CharField(
+        _("Google Analytics code"),
+        max_length=50,
+        blank=True,
+        help_text=_(
+            "Typically looks like 'UA-XXXXX-Y'. Supplying this installs Google Analytics."
+        ),
+    )
+    matomo_url = models.CharField(
+        _("Matomo server URL"),
+        max_length=255,
+        blank=True,
+        help_text=_("The base URL of your Matomo server, e.g. 'matomo.example.com'."),
+    )
+    matomo_site_id = models.PositiveIntegerField(
+        _("Matomo site ID"),
+        blank=True,
+        null=True,
+        help_text=_("The 'idsite' of the website you're tracking in Matomo."),
+    )
+    piwik_url = models.CharField(
+        _("piwik server URL"),
+        max_length=255,
+        blank=True,
+        help_text=_("The base URL of your Piwik server, e.g. 'piwik.example.com'."),
+    )
+    piwik_site_id = models.PositiveIntegerField(
+        _("Piwik site ID"),
+        blank=True,
+        null=True,
+        help_text=_("The 'idsite' of the website you're tracking in Piwik."),
+    )
+    siteimprove_id = models.CharField(
+        _("SiteImprove ID"),
+        max_length=10,
+        blank=True,
+        help_text=_(
+            "Your SiteImprove ID - you can find this from the embed snippet example, "
+            "which should contain a URL like '//siteimproveanalytics.com/js/siteanalyze_XXXXX.js'. "
+            "The XXXXX is your ID."
+        ),
+    )
+
+    analytics_cookie_consent_group = models.ForeignKey(
```

Comment:
Is this intentionally missing from the admin page?

Reply:
nope, oversight!